pylint/utils.py

```diff
@@ -810,7 +815,9 @@ def expand_modules(files_or_modules, black_list, black_list_re):
     result = []
     errors = []
     for something in files_or_modules:
-        if exists(something):
+        if _path_delimiter() in something:
+            modname, filepath = something.split(':')
```

Comment:
@Horta I think `.split` should also be invoked with `path_delimiter` instead of `:`